css/properties/background-clip.json

```diff
@@ -12,7 +12,6 @@
               },
               {
                 "version_added": "1",
-                "version_removed": "64",
```

Comment:
If I had to guess, this was probably due to the alternate synonyms and the fact that Chrome's autocompletion only contains the normal values (which causes issues when trying to test in the developer tools, since it doesn't let you cancel autocompletion).  Now though, that doesn't seem to matter; Chrome is taking the values just fine.

What it looks like is: in prior versions, Chrome *only* accepted the alternate synonyms, but now it accepts the standard ones?

Reply:
I think it's likely that `-webkit-background-clip` and `-webkit-background-origin` originally supported only the "padding", "border" and "content" keywords, but it's so far back that I haven't been able to see that when testing http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=9958. In other words, I haven't been able to confirm the `version_added` here, just disconfirm the `version_removed`.

Reply:
Anything more needed here? If I'm following the conversation right, it sounds like this is OK?

Reply:
No, the data was wrong and now fixed. There might be more to discover about the very early support for the values of `-webkit-background-clip` and `-webkit-background-origin`, but that's not important and just missing information, not any incorrect claim in the data.

Reply:
That's correct -- regardless, I think we can omit researching and adding the older data, since it's for browser versions beyond the two year irrelevance mark, and add it in if someone requests.  This PR is looking good to me!